ibm/resource_ibm_event_streams_topic_test.go

```diff
@@ -0,0 +1,287 @@
+					resource.TestCheckResourceAttr("ibm_event_streams_topic.es_topic", "partitions", strconv.Itoa(partitions)),
+				),
+			},
+			resource.TestStep{
+				Config: testAccCheckIBMEventStreamsTopicWithExistingInstanceWithConfig(existingInstanceName, topicName, partitions, cleanupPolicy, retentionBytes, retentionMs, segmentBytes),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					testAccCheckIBMEventStreamsTopicExists("ibm_event_streams_topic.es_topic", topicName),
+					resource.TestCheckResourceAttrSet("data.ibm_resource_instance.es_instance", "extensions.kafka_brokers_sasl.0"),
+					resource.TestCheckResourceAttrSet("data.ibm_resource_instance.es_instance", "extensions.kafka_http_url"),
+					resource.TestCheckResourceAttrSet("ibm_event_streams_topic.es_topic", "id"),
+					resource.TestCheckResourceAttrSet("ibm_event_streams_topic.es_topic", "kafka_brokers_sasl.0"),
+					resource.TestCheckResourceAttrSet("ibm_event_streams_topic.es_topic", "kafka_http_url"),
+					resource.TestCheckResourceAttr("ibm_event_streams_topic.es_topic", "name", topicName),
+					resource.TestCheckResourceAttr("ibm_event_streams_topic.es_topic", "partitions", strconv.Itoa(partitions)),
+					resource.TestCheckResourceAttr("ibm_event_streams_topic.es_topic", "config.cleanup.policy", cleanupPolicy),
+					resource.TestCheckResourceAttr("ibm_event_streams_topic.es_topic", "config.retention.bytes", strconv.Itoa(retentionBytes)),
+					resource.TestCheckResourceAttr("ibm_event_streams_topic.es_topic", "config.retention.ms", strconv.Itoa(retentionMs)),
+					resource.TestCheckResourceAttr("ibm_event_streams_topic.es_topic", "config.segment.bytes", strconv.Itoa(segmentBytes)),
+				),
+			},
+		},
+	})
+}
+
+func TestAccIBMEventStreamsTopicImport(t *testing.T) {
+	instanceName := fmt.Sprintf("es_instance_%d", acctest.RandInt())
+	planID := "standard"
+	serviceName := "messagehub"
+	location := "us-south"
+	topicName := fmt.Sprintf("es_topic_%d", acctest.RandInt())
+	partitions := 1
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckIBMEventStreamsInstanceDestroy,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccCheckIBMEventStreamsTopicWithoutConfig(instanceName, serviceName, planID, location, topicName, partitions),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					testAccCheckIBMEventStreamsTopicExists("ibm_event_streams_topic.es_topic", topicName),
+					resource.TestCheckResourceAttr("ibm_event_streams_topic.es_topic", "name", topicName),
+					resource.TestCheckResourceAttr("ibm_event_streams_topic.es_topic", "partitions", strconv.Itoa(partitions)),
+				),
+			},
+			resource.TestStep{
+				ResourceName:            "ibm_event_streams_topic.es_topic",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"wait_time_minutes"},
+			},
```

Comment:
ImportStateVerifyIgnore: []string{"wait_time_minutes"}, this is not required we don't have any argument wait_time_minutes in schema

Reply:
I will remove it.